src/test/java/com/terracottatech/frs/io/nio/NIOManagerTest.java

```diff
@@ -349,7 +349,8 @@ public void run() {
         assertNotNull(manager.scan(Constants.FIRST_LSN));
         assertThat(lm.currentLsn(), lessThan(triggerMarker));
         assertThat(latch.getCount(), not(equalTo(0L)));
-        assertThat(loadingThread.getState(), equalTo(Thread.State.WAITING));
+        // race condition on Windows where we can still observe the thread in running state
+        //assertThat(loadingThread.getState(), equalTo(Thread.State.WAITING));
```

Comment:
I think we can just delete these lines. The latch count covers us, and the git commit message can explain why the removal.